doc/object_properties.adoc

```diff
@@ -0,0 +1,265 @@
+= Object Properties
+include::../include/config.adoc[]
+:description: Documentaton of Object Properties
+:keywords: obj, props, object, properties
+:url-c-content-src: {url-engine-src}/script/common/c_content.cpp
+
+WARNING: There may be missing/incorrect information here,
+please refer to the {url-lua-api}#L7141-L7304[Lua API]
+and the {url-c-content-src}#L186-L442[relevant engine code]
+
+WARNING: This is missing the default values for everything
+
+== Properties Applying Only to Players
+
+=== `hp_max`
+* internal type: `u16`
+
+Sets the max HP of a player.
+Clamped if it goes above the `u16` max value or under the `u16` min value.
+
+=== `breath_max`
+* internal type: `u16`
+
+Sets the max breath of a player.
+
+=== `eye_height`
+* internal type: `f32`
+
+How high the camera is in nodes about the player's feet.
+
+=== `zoom_fov`
+* internal type: `f32`
+* units: `degrees`
+
+Sets the field of view of the player when they zoom. Setting to `0` disables zooming.
+
+== Properties Applying to All Objects
+
+=== `physical`
+* field type: `bool`
+
+Controls whether the object collides with nodes that are `walkable`.
+
+=== `collide_with_objects`
+* field type: `bool`
+
+Controls whether the object collides with other objects.
+
+CAUTION: `physical` must be set to `true` for this to take effect!
```

Comment:
Perhaps also worth noting that this will work for players as well, but exhibits strange bugs (such as this one: https://github.com/minetest/minetest/issues/11783)

Reply:
is this not implied by the `Properties Applying to All Objects` subheading?

Reply:
> is this not implied by the `Properties Applying to All Objects` subheading?

The former yes, the latter no.